src/shared/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -53,19 +53,17 @@ const VideoPlayer: React.ForwardRefRenderFunction<HTMLVideoElement, VideoPlayerP
       return
     }
 
-    if (playing != null) {
```

Comment:
Changes here are out of scope of this PR, feel free to open a new one with some rationale